packages/p2p/src/components/separator-container-line/separator-container-line.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type TSeparatorContainerLineProps = {
+    className?: string;
+    is_invisible?: boolean;
+};
+
+const SeparatorContainerLine = ({ className, is_invisible = false }: TSeparatorContainerLineProps) => (
```

Comment:
are we using this is_invisible prop in any of the components that use separator-container-line?

Reply:
not really but I kept it there as I thought it the future it would be used, there mustve been some purpose 😆

Reply:
i think we can remove it now itself.. if not, then we will remove it only in the next refactoring which might be after a long time..